routes/movies.routes.js

```diff
@@ -0,0 +1,71 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+const Movie = require('../models/Movie.model');
+
+
+// Iteration 6: Create Movies
+// GET
+router.get("/movies/create", (req,res,next) => {
+    Movie.find()
```

Comment:
Here instead of querying for the movies you would want to query for the celebrities, as this is what needs to be populated when you create a new movie

```
   Celebrity.find()
    .then(celebrities => {
        res.render("movies/new-movie", { celebrities }); 
    })
```